packages/odyssey-react-mui/README.md

```diff
@@ -47,11 +47,36 @@ import { PasswordInput } from "@okta/odyssey-react-mui";
 Add the Material-UI `ThemeProvider` around your whole app:
 
 ```jsx
-import { ThemeProvider } from "@mui/material/styles";
-import { odysseyTheme } from "@okta/odyssey-react-mui";
+import { ThemeProvider } from "@okta/odyssey-react-mui";
+
+const YourAppRoot = ({ children }) => <ThemeProvider>{children}</ThemeProvider>;
+```
+
+Add Material-UI components not exported from Odyssey with Odyssey styling:
+
+```jsx
+import { ThemeProvider as MuiThemeProvider } from "@mui/material/styles";
+import { odysseyTheme, ThemeProvider } from "@okta/odyssey-react-mui";
+
+const YourAppRoot = ({ children }) => (
+  <MuiThemeProvider theme={odysseyTheme}>
```

Comment:
Curious: why do we need both muithemeprovider and theme provider from odyssey-react-mui here ?

Reply:
I don't want consumers to have to worry about this, but if they need access to MUI's theme provider, they still have it available.

As far as I understand, MUI's theme provider is accumulative. So if you override 1 prop, it merges the previous theme values in context and only overrides that 1 prop.

We don't give access to extend `odysseyTheme`. It's created by MUI's `createTheme()` and then exported. That's why there's a separate `ThemeProvider`. It's meant to simplify it. The `MuiThemeProvider` is unnecessary except for extreme use cases.